src/hicFile.js

```diff
@@ -1,3 +1,4 @@
+import {Alert} from "../node_modules/igv-ui/dist/igv-ui.js"
```

Comment:
just a watcher of this repo, but it might be good if this data parsing module maybe did not incorporate alert ui, even as an optional situation e.g. with args.parentElement. that would make it more pure and avoid it being bundled in by bundlers

Reply:
Yes agreed @cmdcolin ,  we don't want that dependency here.

Reply:
Agreed. npm module igv-ui dependancy is gone via latest commit.